Library/Homebrew/cmd/update.sh

```diff
@@ -220,8 +220,15 @@ homebrew-update() {
   for option in "$@"
   do
     case "$option" in
-      # TODO: - `brew update --help` should display update subcommand help
-      --help) brew --help; exit $? ;;
+      --help)
+        sed -n '/^  \* `update`/,/^  \* / {
+          s/`//g;               # Strip out backticks
+          s/\* update/update/;  # Mark the headers we want to keep
+          /\*/d;                # and delete the others
+          s/^  //;              # Adjust the spacing a bit
+          p;
+        }' "$HOMEBREW_LIBRARY/Homebrew/manpages/brew.1.md"
+      exit 0 ;;
```

Comment:
Yeah, I guess, but this is a second faster, not even kidding. So it had some benefit. But if we move the help test into the files them selfs, this definitely needs to go.